exercises/dominoes/dominoes_test.py

```diff
@@ -0,0 +1,114 @@
+import unittest
+
+from dominoes import chain
+
+
+# test cases adapted from `x-common//canonical-data.json` @ version: 1.0.0
```

Comment:
There's a new format for the version string to reflect the name-change from `x-common` to `problem-specifications` that happened a while back (#784).

```
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Reply:
I actually noticed that these tests weren't even based off of 1.0.0; it was the current version, 2.0.0. This has been updated.